source/numeric/SVInt.cpp

```diff
@@ -637,25 +637,27 @@ std::ostream& operator<<(std::ostream& os, const SVInt& rhs) {
     return os;
 }
 
-std::string SVInt::toString() const {
+std::string SVInt::toString(bitwidth_t abbreviateThresholdBits, bool exactUnknowns) const {
     // guess the base to use
     LiteralBase base;
-    if (bitWidth < 8 || (unknownFlag && bitWidth <= 64))
+    // unless the int is wholly or digit/hexit-wise X or Z (TODO: worth checking?),
```

Comment:
Probably not worth the effort. I'd just remove the TODO.

Reply:
Done